.github/workflows/e2e_nightly.yml

```diff
@@ -1,17 +1,19 @@
 name: E2E Nightly
 
-on:
-  schedule:
-    # Run every day at 23:00 UTC
-    - cron: '0 23 * * *'
+on: [push, pull_request]
+#on:
+#  schedule:
+#    # Run every day at 23:00 UTC
+#    - cron: '0 23 * * *'
 
 jobs:
   e2e-build-hw:
     name: Build SYCL, UR, run E2E
     strategy:
       matrix:
         adapter: [
```

Comment:
you could add `fail-fast: false` - we may want to run all jobs independently (at least, as long as they run in nightly job)

Reply:
I think it should fail to not block other runners if anything is failing.